controllers/policyendpoints_controller.go

```diff
@@ -211,11 +238,10 @@ func (r *PolicyEndpointsReconciler) updatePolicyEnforcementStatusForPods(ctx con
 		deletePinPath := true
 		podIdentifier := utils.GetPodIdentifier(targetPod.Name, targetPod.Namespace, r.log)
```

Comment:
Do we still need podIdentifier? Seems like only used in the below log?

Reply:
Yeah we can remove that one. I will add optimization and clean up of this line in next PR